demo.py

```diff
@@ -68,7 +73,9 @@ def main():
   })
 
   # Sign and dump layout
-  layout.sign(alice_key)
+  print "Load alice private key to sign layout..."
+  alice_private = toto.util.prompt_import_rsa_key_from_file("alice")
+  layout.sign(alice_private)
```

Comment:
Internally we keep using the key dictionary. I just changed the way how keys are stored. Therefor no changes in the models are required.